articles/030_ros_with_zeromq.md

```diff
@@ -18,7 +18,7 @@ author: '[William Woodall](https://github.com/wjwwood)'
 {{ page.abstract }}
 </div>
 
-> **This document pre-dates the decision to build ROS 2 on top of DDS.**
```

Comment:
This warning and the one in serialization.md seem important enough to keep the emphasis/boldface formatting. Although, they do come at the beginning of the document, so I guess they're already emphasized.